dashboards-observability/public/components/visualizations/charts/logs_view/logs_view_type.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { LogsView } from './logs_view';
+import { getPlotlyCategory, getPlotlySharedConfigs } from '../shared/shared_configs';
+import { LensIconChartDatatable } from '../../assets/chart_datatable';
+import { VizDataPanel } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/default_vis_editor';
+import { PLOTLY_COLOR } from '../../../../../common/constants/shared';
+import { ConfigLogsView } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_logs_view';
+import { ButtonGroupItem } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_button_group';
+import { ConfigSwitch } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_switch';
+import { InputFieldItem } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls';
+
+const sharedConfigs = getPlotlySharedConfigs();
+const VIS_CATEGORY = getPlotlyCategory();
+
+export const createLogsViewTypeDefinition = (params: any = {}) => ({
+  name: 'logs_view',
+  type: 'logs_view',
+  id: 'logs_view',
+  label: 'Logs View',
+  fullLabel: 'Logs View',
```

Comment:
significance of full label over label?

Reply:
It is used as per the interface `IVisualizationTypeDefination` designed previously.